react/features/base/media/middleware.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 
-import { SET_ROOM } from '../conference';
+import { SET_ROOM, setAudioOnly } from '../conference';
```

Comment:
Technically, the `audioOnly` state belongs to base/conference, not base/media. So I personally expect to find its initialization from the `locationURL` and config.js in base/conference, not base/media. Practically, I presume it was easier to write the source code here because it looks like `config.startWithAudioMuted` and `config.startWithVideoMuted`.

Reply:
Yep. Also, I think we might need to reconsider the audioOnly state as base/media, since it's an intention we have and then apply...